Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -2061,10 +2061,22 @@ private static void ReportFileAccesses(ExecutionResult processExecutionResult, F
                     Contract.Assert(pipCacheMiss.Value.CacheMissType != PipCacheMissType.Invalid, "Must have valid cache miss reason");
                     environment.Counters.IncrementCounter((PipExecutorCounter)pipCacheMiss.Value.CacheMissType);
 
-                    Logger.Log.ScheduleProcessPipCacheMiss(
-                        operationContext,
-                        cacheableProcess.Description,
-                        runnableFromCacheResult.Fingerprint.ToString());
+                    if (processRunnable.Process.Weight > 1)
+                    {
+                        Logger.Log.ProcessPipCacheMissWithProcessWeight(
+                            operationContext,
+                            cacheableProcess.Description,
+                            runnableFromCacheResult.Fingerprint.ToString(),
+                            processRunnable.Process.Weight.ToString());

```

Comment:
I do not like logging this when doing the cache lookup. in distributedbuilds, we use different workers for cache lookup and execute. I'd like to see the weight of the process in the worker where we execute the pip

Reply:
Do you know of a good execution time message for me to add to? Otherwise I can just add a brand new one, but it seemed wasteful.

I was also thinking of adding a field to critical path logging, only if process weight is > 1

---
In reply to: [271480547](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271480547) [](ancestors = 271480547)

Reply:
Ok I made a new log message that logs at executeprocess time. Decided not to append to critical path info for now.

---
In reply to: [271481873](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271481873) [](ancestors = 271481873,271480547)

Reply:
sounds good!

---
In reply to: [271481873](https://github.com/Microsoft/BuildXL/pull/67#discussion_r271481873) [](ancestors = 271481873,271480547)